tree/tree/src/TBasket.cxx

```diff
@@ -699,6 +699,26 @@ Int_t TBasket::ReadBasketBytes(Long64_t pos, TFile *file)
    return fNbytes;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Disown all references to the internal buffer - some other object likely now
+/// owns it.
+///
+/// This TBasket is now useless and invalid until it is told to adopt a buffer.
+void TBasket::DisownBuffer()
+{
+   fBufferRef = NULL;
```

Comment:
What if this is called when the TBaskets owns the fBufferRef?

Reply:
Really, this is part of the contractual relationship between TBranch and TBasket.  Will make this into a private method.